src/main/java/VASSAL/tools/ipc/IPCMessageReceiver.java

```diff
@@ -38,7 +38,8 @@ public void run() {
 //System.err.println("received " + msg);
 
         lsup.notify(msg);
-      } while (!(msg instanceof Fin));
+      }
+      while (!(msg instanceof Fin));
```

Comment:
I thought you said you were reverting this change?

Reply:
Missed that one, will revert it.